### DISCUSSION TITLE
How to insert newline in table cell

### DISCUSSION THREAD

**Discussion by Hokwang** [apache/superset#23473] [Q&A / Help]
Hi,

I usually use table in superset.

I want to add newline in cell.

For now, I use below code,
`(SELECT string_agg(t->>'GroupName', E'\n') FROM jsonb_array_elements(s`
(my db is postgresql)

and it shows. This is not right newline.
![image](https://user-images.githubusercontent.com/1247030/227240897-8d494eaf-4e86-4502-97c6-f87d13485df9.png)

Is there a way to add newline?

**Comment by Volounteer** [comment]
Hello! Try this delimiter ` <br>`:
`SELECT string_agg(t->>'GroupName', '<br>') FROM ...`

Also, make sure to enable the 'Render columns in HTML format' option in your table's customization settings.
